README.md

```diff
@@ -1,185 +1,212 @@
+3. Click **Upload app**.
+4. In the **From this URL field**, enter: %s%s
+5. Wait for the app to install. Once completed, you should see an "Installed and ready to go!" message.
+6. Use the "/jira connect" command to connect your Mattermost account with your Jira account.
+7. Click the "More Actions" (...) option of any message in the channel (available when you hover over a message).
+
+If you see an option to create a Jira issue, you're all set! If not, see our [Frequently Asked Questions](#5-frequently-asked-questions-faq) for troubleshooting help.
 
-#### 3.2.3 Jira - Deploy via Jira Application Links
+##### Step 3: Jira Server or Data Center
 
-1. In Mattermost, post a command `/jira instance add server <your-jira-server-url>`. This generates the consumer key and public key used on a later step.
-2. As a Jira System Administrator, go to **Jira Settings > Applications > Application Links**.
-3. Enter your Mattermost URL as the application link, then click **Create new link**.
-4. In **Configure Application URL** screen, confirm your Mattermost URL is included as the application URL. Ignore any displayed errors and click **Continue**.
-5. In **Link Applications** screen, set the following values:
+As a Mattermost System Admin, post `/jira install server <your-jira-url>`, and follow the steps posted to the channel. They are also outlined below:
+
+1. As a Jira System Administrator, go to **Jira Settings > Applications > Application Links**.
+2. Enter your Mattermost URL as the application link, then click **Create new link**.
+3. In **Configure Application URL** screen, confirm your Mattermost URL is entered as the "New URL". Ignore any displayed errors and click **Continue**.
+4. In **Link Applications** screen, set the following values:
   - **Application Name**: Mattermost
   - **Application Type**: Generic Application
-6. Check the **Create incoming link** value, then click **Continue**.
-7. In the following **Link Applications** screen, set the following values:
+5. Check the **Create incoming link** value, then click **Continue**.
+6. In the following **Link Applications** screen, set the following values:
   - **Consumer Key**: Copy the value generated in step 1 for this field.
   - **Consumer Name**: Mattermost
   - **Public Key**: Copy the value generated in step 1 for this field.
-8. Click **Continue**.
+7. Click **Continue**.
+8. Use the "/jira connect" command to connect your Mattermost account with your Jira account.
+9. Click the "More Actions" (...) option of any message in the channel (available when you hover over a message).
 
-You're all set. Users can now connect their Mattermost account with Jira using `/jira connect`.
+If you see an option to create a Jira issue, you're all set! If not, see our [Frequently Asked Questions](#5-frequently-asked-questions-faq) for troubleshooting help.
 
-#### 3.1.4 Jira - Configure Webhooks
+## 3. Jira v2 Roadmap
```

Comment:
Keeping this, because people keep asking me about roadmap timelines and scheduled features.

Will be removed once v2.1 is ready.

Reply:
Just a comment - I like this format and wondering if we should consider adding a similar format to other feature documentation even if the improvements are not scheduled yet.  Something to consider for Season of Docs -feature template project.